tools/test/config/config_test.py

```diff
@@ -252,16 +252,3 @@ def test_parameters_and_config_macros_to_macros():
 
     macro_list = Config._parameters_and_config_macros_to_macros(params, macros)
     assert macro_list == ["CUSTOM_MACRO_NAME=1"]
-
-
-@pytest.mark.parametrize("target_start_size", [
-    ("FUTURE_SEQUANA_PSA", 0x10080000, 0x78000),
```

Comment:
instead of removing the test entirely, modify it to use CY8CKIT_062_WIFI_BT_M0_PSA and CY8CKIT_062_WIFI_BT_PSA
that way we won't lose coverage

Reply:
@orenc17 done